### PR TITLE
complete the transition to automatically built cheatsheets

### PR DESCRIPTION
**Pull request by jimustafa**
Now that the cheatsheets and handouts are being automatically built (#57) and published (#63), it is probably unnecessary to track them in the repository. This PR removes the out-of-date cheatsheets and handouts, and updates the `README` to reflect the new location of the up-to-date versions at https://matplotlib.org/cheatsheets/.

It would be nice to automatically add a directory listing when building the artifacts and committing to `gh-pages` so that the location of the cheatsheets and handouts could be given as https://matplotlib.org/cheatsheets/ without it yielding a `404`. Some possible solutions are discussed [here](https://stackoverflow.com/q/39048654).

### CODE REVIEW

**Comment by rougier** [comment]
Timeout on some url during the check. Can you do a dummy commit to restart the check?

**Comment by QuLogic** [comment]
You can re-run the CI from the Checks page (which I've just done.)

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# built cheatsheets and handouts
+# ----------------------------------
+cheatsheets*.p[dn][fg]
+handout-*.p[dn][fg]
```
Comment (QuLogic) [inline .gitignore:4]: This also hits ,pdg and .pnf, which probably don't exist, but better to be explicit
```suggestion
cheatsheets*.pdf
cheatsheets*.png
handout-*.pdf
handout-*.png
```

Comment (jimustafa) [reply .gitignore]: Agreed, better to be explicit. This has been fixed.

**Pull request merged**